### DISCUSSION TITLE
Meepctl dockerize all fialed

### DISCUSSION THREAD

**Discussion by sunilaradhya** [InterDigitalInc/AdvantEDGE#435] [Q&A] [answered]
Hi, 
When I run meepctl dockerize all command it failed to push meep-docker-registry

Please help to resolve this.
Also please confirm except system any hardware equipment/device required. I am using Ubuntu systems with specified system requirements as mentioned in AdvantEDGE document. Also please confirm this AdvantEDGE run or deploy on Non-root user only.pls reply with solution.

**Comment by dilallkx** [comment, marked as answer]
Hi @sunilaradhya,

Thank you for reaching out!

For the meepctl dockerize failure, this typically occurs if the dependency pods have not been deployed (e.g. meep-docker-registry) or if the k8s certificate authority has not been added to the trusted CA list. Please make sure to run [step 7 of the runtime setup](https://interdigitalinc.github.io/AdvantEDGE/docs/setup/env-runtime/#step-7---configure-docker-registry) and redeploy the dependency pods. Please let me know if this fixes your issue.

WRT hardware requirements, they are specified [here](https://interdigitalinc.github.io/AdvantEDGE/docs/setup/env-hw/#hardware-requirements); no additional hardware required.

WRT not-root user, this should work; any steps in the procedure that require sudo/root privileges are explicitly stated.

I hope this helps.
Regards!

**Reply by sunilaradhya** [reply]
Hi @dilallkx 
Thanks for your reply. I deployed dependency pods on non root user. Dependency pods deployed and pods are running fine. When I deployed core pods core pods except ingress others pods are in crashlooopBACKOFF and also meep-webhook pod showing as init 0/1 state.
Please help to resolve the issue. 

Regards
Sunil
